ssniffer/Makefile

```diff
@@ -1,8 +1,28 @@
+prefix = /usr/local
+CP= cp
+RM= rm -f
+INSTALL= install -m755 -D
+TARGET=ssniffer
 
+CFLAGS += -Wall -O2
 
-FLAGS= -Wall -O2
+all: $(TARGET)
+
+$(TARGET): $(TARGET).c 
+	@echo "CFLAGS=$(CFLAGS)" | \
```

Comment:
Why show the CFLAGS variable?

Reply:
In order to verify the modifications made by the debian utilities, we can delete if you want.